homeassistant/components/tplink/manifest.json

```diff
@@ -3,6 +3,6 @@
   "name": "TP-Link Kasa Smart",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/tplink",
-  "requirements": ["pyHS100==0.3.5"],
```

Comment:
I really want to get pyHS100 out of Home Assistant as it is the last dependency that depends on `typing` and so prevents us from blocking `typing` from being installed, something that breaks new venv installations and CI unless we put extra code to fix it.

Reply:
This has been resolved.